### PR TITLE
MON-3673: Bump Prometheus to v2.49.1

### PR DESCRIPTION
**Pull request by simonpasquier**
Some conflicts needed to be fixed manually.

### CODE REVIEW

**Comment by danielmellado** [comment]
lgtm, let's wait for CI

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 <!--
-    OpenShift: Don't forget to run `./scripts/rh-manifest.sh` from the
```
Comment (machine424) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: is this intentional?

Comment (machine424) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: I asked ART if we can get rid of that manifest.

Comment (danielmellado) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: Good catch there! Once we can cofirm that the cachito integration is done from their side we can get rid of it

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -6,3 +6,5 @@
 /tsdb @jesusvazquez
 /promql @roidelapluie
 /cmd/promtool @dgl
+/documentation/prometheus-mixin @metalmatze
+
```
Comment (danielmellado) [inline .github/CODEOWNERS:10]: Could we get rid of the space? It does seem to complain

Comment (simonpasquier) [reply .github/CODEOWNERS:10]: this is to be fixed upstream then

Comment (danielmellado) [reply .github/CODEOWNERS:10]: oh, actually it does seem to complain about ```Unknown owner on line 1: make sure @foo exists and has write access to the repository
```
So I'm wondering whether we should just keep that or override directly.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (simonpasquier) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: nope, will fix it

**Pull request merged**